docs/how-to/add-delete-user-entity.md

```diff
@@ -20,7 +20,7 @@ However, you can choose to use your own custom Authentication entity as long as
 
 ## Dependencies Between the Authentication Entity and Authentication Plugins
 
-The authentication plugins, such as the _NestJS Auth Module_ and others, rely on the existence of an _Authentication_ entity to handle user authentication and authorization.
+The authentication plugins rely on the existence of an _Authentication_ entity to handle user authentication and authorization. For Node.js services, this includes the _NestJS Auth Module_ and others. For .NET services, this includes the _ASP.NET Core Identity_ plugin.
```

Comment:
I think it should be "For Node.js services, this includes the _NestJS Auth Module_ or others".
I.e. "or" instead of "and". Is that correct- English-wise?